examples/boids/main.rs

```diff
@@ -193,16 +193,16 @@ impl framework::Example for Example {
         for i in 0..2 {
             particle_bind_groups.push(device.create_bind_group(&wgpu::BindGroupDescriptor {
                 layout: &compute_bind_group_layout,
-                bindings: &[
-                    wgpu::Binding {
+                entries: &[
+                    wgpu::BindGroupEntry {
                         binding: 0,
                         resource: wgpu::BindingResource::Buffer(sim_param_buffer.slice(..)),
                     },
```

Comment:
I've renamed `bindings` to `entries` and `Binding` to `BindGroupEntry` for consistency with `wgpu-core`. This will cause some breakage unfortunately.

Reply:
thank you!